src/components/policy/policy_external/src/sql_pt_representation.cc

```diff
@@ -649,10 +662,29 @@ bool SQLPTRepresentation::GatherFunctionalGroupings(
         }
       }
     }
+
+    rpcs.Reset();
```

Comment:
@LuxoftAKutsan Looks better to me, that was the reason. Probably more important question would be: does this change make any functional difference. Looks like not, could you please confirm.